packages/manager/apps/iplb/package.json

```diff
@@ -54,7 +54,7 @@
     "chart.js": "chartjs/Chart.js#^2.0",
     "flatpickr": "~4.5.2",
     "jquery": "^2.1.3",
-    "moment": "^2.19",
+    "moment": "^2.24.0",
```

Comment:
No need to change the import or the webpack config for standalone apps ?

Reply:
You're right! We have to find solutions for standalone applications too.

As we were mainly focused on the "Universe" (Large) applications I've only updated the impacted webpack configurations.

Last weeks some discussions has started around a shift to date-fns or other libraries.

We can keep investigating to a suitable solution for the long-term and then apply it across all applications.